platform/qt/src/qmapboxgl.cpp

```diff
@@ -1062,6 +1314,15 @@ void QMapboxGL::setFilter(const QString& layer_, const QVariant& filter_)
     qWarning() << "Layer doesn't support filters";
 }
 
+/*!
+    Renders the map using OpenGL draw calls. If \a fbo is passed, it will
+    make sure to bind the framebuffer object before drawing, otherwise a
+    valid GL context is expected with an appropriated GL viewport state set
+    for the size of the canvas.
+
+    This function should be called only after the signal needsRendering() is
```

Comment:
Sorry, was referring to the paragraph as a whole, not the reference to `needsRendering()`. In Doxygen syntax, [`\pre`](https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdpre) means “precondition” and given special prominence.